ui/tests/integration/breadcrumb-trail-test.js

```diff
@@ -0,0 +1,41 @@
+import { findAll, find } from 'ember-native-dom-helpers';
+import { test, moduleForComponent } from 'ember-qunit';
+import { faker } from 'ember-cli-mirage';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('breadcrumb-trail', 'Integration | Component | breadcrumb trail', {
+  integration: true,
+});
+
+
+test('component renders correctly', function(assert) {
+  const fixtures = createFixtures();
+  this.set(
+    'items',
+    fixtures
+  );
+  this.render(hbs`
+    {{breadcrumb-trail items=items}}
+  `);
+
+  assert.equal(findAll('nav').length, 1, 'nav exists');
+  assert.equal(findAll('a').length, fixtures.length, `There are ${fixtures.length} breadcrumbs`);
+  assert.equal(find('[data-test-breadcrumb="0"]').textContent, fixtures[0].label, 'The first breadcrumb has the correct label');
+});
```

Comment:
This is a great first test for the component. I think a couple more important things to test are active class (impossible right now since it depends on document.location), label rendering, and href building. Unfortunately I think href building is also currently impossible to build since it depends on the router which isn't present in integration tests.